### PR TITLE
Progressively loads team profile images

### PR DESCRIPTION
**Pull request by Codebmk**
#### Summary of Changes (What does this PR do?)
- Loads images on scroll to the viewport. Once the image has entered the viewport, it is displayed.
#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [x] I consider this code done
- [ ] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* git pull origin staging
* git checkout WEB-img-lazy-loading
* set the following environment variables
```
 DEBUG=True
 SECURE_SSL_REDIRECT=False
 HTTP_X_FORWARDED_PROTO=http
 CORS_ORIGIN_ALLOW_ALL=True
 CSRF_TRUSTED_ORIGINS="http://127.0.0.1:8000"
```
* run `python manage.py runserver` and `inv webpack-server` concurrently

#### What are the relevant tickets?
- [WEB-257](https://airqoteam.atlassian.net/browse/WEB-257)

#### [**PR Demo link**](https://drive.google.com/file/d/1nPR1pHQdeH3nnS1gYnoUaSfZFPL2stAC/view?usp=sharing)
team

### CODE REVIEW

**Review by Baalmart — COMMENTED** [review]
PR looks neat to me, I will let @FaithDaka first test it out locally on her machine. After her approval, we should be good to go. Thanks @Codebmk

File: website/backend/settings.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
 For the full list of settings and their values, see
 https://docs.djangoproject.com/en/3.1/ref/settings/
 """
+from email.policy import default
```
[the remark this answers is not in the document]
Comment (Codebmk) [reply website/backend/settings.py:12]: Hmm I didn't add that.

[the remark this answers is not in the document]
Comment (Codebmk) [reply website/backend/settings.py:12]: Gitlens says you added it. Looks like it is not used with in the settings.py

[the remark this answers is not in the document]
Comment (FaithDaka) [reply website/backend/settings.py:12]: Must have been an auto add from the 'default' word. Have been able to remove it?

[the remark this answers is not in the document]
Comment (Codebmk) [reply website/backend/settings.py]: Done!

**Review by FaithDaka — APPROVED** [review]

**Review by Baalmart — APPROVED** [review]
Thanks @Codebmk

**Pull request merged**